### DISCUSSION TITLE
Fosscord

### DISCUSSION THREAD

**Discussion by pikachub2005** [discordjs/discord.js#7582] [Q&A] [answered]
Fosscord is something I found recently, and apparently they're going to connect Discord accounts. Would those count as selfbots, and would my account get deleted if I use it?

**Comment by vladfrangu** [comment, marked as answer]
We don't know what Fosscord is, but anything that connects a user account to Discord that isn't their official clients counts as client modding/selfbotting and is against the Terms of Service

**Reply by almostSouji** [reply]
To add: ask policy questions to Discord, not us: <https://support.discord.com>
ToS: <https://discord.com/terms>, <https://discord.com/guidelines>

> Do not use self-bots or user-bots. Each account must be associated with a human, not a bot. Self-bots put strain on Discord’s infrastructure and our ability to run our services. For more information, you can read our Developer Policies [here](https://discord.com/developers/docs/policy).

**Reply by pikachub2005** [reply]
Ok, thanks
